tests/integrate_test/cita_basic.sh

```diff
@@ -1,117 +1,97 @@
 
-################################################################################
-echo -n "2) start nodes  ...  "
-for i in {0..3} ; do
-    bin/cita bebop setup node/$i  > /dev/null
-done
-for i in {0..3} ; do
-    bin/cita bebop start node/$i trace > /dev/null &
-done
-echo "DONE"
+    echo -n "2) start nodes  ...  "
+    start_nodes
+    echo "DONE"
 
-################################################################################
-echo -n "3) check height growth normal  ...  "
-timeout=$(check_height_growth_normal 0 60)||(echo "FAILED"
-                                            echo "error msg: ${timeout}"
-                                            exit 1)
-echo "${timeout}s DONE"
+    echo -n "3) check height growth normal  ...  "
+    timeout=$(check_height_growth_normal 0 60)||(echo "FAILED"
+                                                echo "error msg: ${timeout}"
+                                                exit 1)
+    echo "${timeout}s DONE"
 
-################################################################################
-echo -n "4) stop node3, check height growth  ...  "
-bin/cita bebop stop node/3 > /dev/null
-timeout=$(check_height_growth_normal 0 15) || (echo "FAILED"
-                                               echo "error msg: ${timeout}"
-                                               exit 1)
-echo "${timeout}s DONE"
+    echo -n "4) stop node3, check height growth  ...  "
+    bin/cita bebop stop node/3 > /dev/null
+    timeout=$(check_height_growth_normal 0 30) || (echo "FAILED"
+                                                   echo "error msg: ${timeout}"
+                                                   exit 1)
+    echo "${timeout}s DONE"
 
-################################################################################
-echo -n "5) stop node2, check height stopped  ...  "
-bin/cita bebop stop node/2 > /dev/null
-timeout=$(check_height_stopped 0 27) || (echo "FAILED"
-                                         echo "error msg: ${timeout}"
-                                         exit 1)
-echo "${timeout}s DONE"
+    echo -n "5) stop node2, check height stopped  ...  "
+    bin/cita bebop stop node/2 > /dev/null
+    timeout=$(check_height_stopped 0 30) || (echo "FAILED"
+                                             echo "error msg: ${timeout}"
+                                             exit 1)
+    echo "${timeout}s DONE"
 
-################################################################################
-echo -n "6) start node2, check height growth  ...  "
-bin/cita bebop start node/2 trace > /dev/null &
-timeout=$(check_height_growth_normal 0 60) || (echo "FAILED"
-                                               echo "error msg: ${timeout}"
-                                               exit 1)
-echo "${timeout}s DONE"
+    echo -n "6) start node2, check height growth  ...  "
+    bin/cita bebop start node/2 trace > /dev/null &
+    timeout=$(check_height_growth_normal 0 60) || (echo "FAILED"
```

Comment:
Is 60s enough? Recommend 90s.

Reply:
i don't change this value.
Just add a main function and make the tls test as a part of this script.

Reply:
Okay!